conf/application.conf

```diff
@@ -27,6 +27,11 @@ play.http.filters = "uk.gov.hmrc.play.bootstrap.filters.FrontendFilters"
 
 play.application.loader = "uk.gov.hmrc.play.bootstrap.ApplicationLoader"
 
+play.filters.enabled += "uk.gov.hmrc.play.bootstrap.frontend.filters.SessionIdFilter"
```

Comment:
in order to use play.filters.enabled we need to remove the line : 

play.http.filters = "uk.gov.hmrc.play.bootstrap.filters.FrontendFilters"

We had done this, not sure how it creeped back in